server/cloudspanner_storage_provider.go

```diff
@@ -31,15 +32,17 @@ import (
 )
 
 var (
-	csURI                  = flag.String("cloudspanner_uri", "", "Connection URI for CloudSpanner database")
-	csNumChannels          = flag.Int("cloudspanner_num_channels", 0, "Number of gRPC channels to use to talk to CloudSpanner.")
-	csSessionMaxOpened     = flag.Uint64("cloudspanner_max_open_sessions", 0, "Max open sessions.")
-	csSessionMinOpened     = flag.Uint64("cloudspanner_min_open_sessions", 0, "Min open sessions.")
-	csSessionMaxIdle       = flag.Uint64("cloudspanner_max_idle_sessions", 0, "Max idle sessions.")
-	csSessionMaxBurst      = flag.Uint64("cloudspanner_max_burst_sessions", 0, "Max concurrent create session requests.")
-	csSessionWriteSessions = flag.Float64("cloudspanner_write_sessions", 0, "Fraction of write capable sessions to maintain.")
-	csSessionHCWorkers     = flag.Int("cloudspanner_num_healthcheckers", 0, "Number of health check workers for Spanner session pool.")
-	csSessionHCInterval    = flag.Duration("cloudspanner_healthcheck_interval", 0, "Interval betweek pinging sessions.")
+	csURI                                = flag.String("cloudspanner_uri", "", "Connection URI for CloudSpanner database")
+	csNumChannels                        = flag.Int("cloudspanner_num_channels", 0, "Number of gRPC channels to use to talk to CloudSpanner.")
+	csSessionMaxOpened                   = flag.Uint64("cloudspanner_max_open_sessions", 0, "Max open sessions.")
+	csSessionMinOpened                   = flag.Uint64("cloudspanner_min_open_sessions", 0, "Min open sessions.")
+	csSessionMaxIdle                     = flag.Uint64("cloudspanner_max_idle_sessions", 0, "Max idle sessions.")
+	csSessionMaxBurst                    = flag.Uint64("cloudspanner_max_burst_sessions", 0, "Max concurrent create session requests.")
+	csSessionWriteSessions               = flag.Float64("cloudspanner_write_sessions", 0, "Fraction of write capable sessions to maintain.")
+	csSessionHCWorkers                   = flag.Int("cloudspanner_num_healthcheckers", 0, "Number of health check workers for Spanner session pool.")
+	csSessionHCInterval                  = flag.Duration("cloudspanner_healthcheck_interval", 0, "Interval betweek pinging sessions.")
+	csDequeueAcrossMerkleBucketsFraction = flag.Float64("cloudspanner_dequeue_bucket_fraction", 0.75, "Fraction of merkle keyspace to dequeue from, set to zero to disable.")
+	csReadOnlyStaleness                  = flag.Duration("cloudspanner_readonly_staleness", time.Minute, "How far in the past to perform readonly operations. Within limits, raising this will help to increase performance.")
```

Comment:
That's a bold claim! Seriously I suggest "will" -> "could" and mention latency impact?

Reply:
Done